business_objects/comments.py

```diff
@@ -80,7 +81,17 @@ def get_add_info_category(
         """
     else:
         table_name = category.get_table_name()
-        name_col = category.get_name_col()
+
+        if category == enums.CommentCategory.RECORD:
+            columns = record.get_first_no_text_column(project_id, xfkey)
+            data = columns[0]
+            no_text_column = columns[1]
+            name_col = (
+                "'" + str(no_text_column) + ": " + str(data[no_text_column]) + "'"
+            )
```

Comment:
i really like the idea of using a constant as a name col. to optimize a bit I would grab the value from the database directly (ill list the query further down)
```suggestion
            name_col = record.get_first_no_text_value(project_id,xfkey)
```